### PR TITLE
feat: add auxiliary libraries block

### PR DESCRIPTION
**Pull request by nikvoblikov**
**This PR:**
Added new auxiliary libraries block to Infrastructure page

**Steps to test**

- Open the [Infrastructure page](https://deploy-preview-316--ebpfio.netlify.app/infrastructure)
- Confirm that everything looks and works as expected

### CODE REVIEW

**Comment by xmulligan** [comment]
It should link to https://github.com/xdp-project/xdp-tools and please remove it from the libraries above

**Comment by nikvoblikov** [comment]
Okey, thank you for comments.

File: src/common/projects/Libraries.js [hunk 1]
```diff
@@ -73,7 +68,7 @@ const auxiliaryLibraries = [
     title: 'Utilities for use with XD',
```
Comment (xmulligan) [inline src/common/projects/Libraries.js:68]:
```suggestion
    title: 'Utilities for use with XDP',
```

File: src/common/projects/Libraries.js [hunk 2]
```diff
@@ -99,16 +110,58 @@ const LibraryCard = ({name, logo, libraries}) => {
...
+        <div className='project-description' dangerouslySetInnerHTML={{__html: description}}></div>
+        {urls && <div className="project-links-container">
+          {urls.map(({label, url}, index) => (
+            <a
+              className="project-link"
+              href={url}
+              target="_blank"
+              key={index}
+            >
+              <img src={icons[label]} width='26px' height='30px'/>
```
Comment (vannyle) [inline src/common/projects/Libraries.js:138]:
```suggestion
              <img src={icons[label]} width={26} height={30} />
```

File: src/common/projects/Libraries.js [hunk 3]
```diff
@@ -99,16 +110,58 @@ const LibraryCard = ({name, logo, libraries}) => {
...
+    </div>
+    <div className='project-inner'>
+      <div className={cn("auxiliary-libraries-body")}>
+        <div className='project-description' dangerouslySetInnerHTML={{__html: description}}></div>
+        {urls && <div className="project-links-container">
+          {urls.map(({label, url}, index) => (
+            <a
+              className="project-link"
+              href={url}
+              target="_blank"
```
Comment (vannyle) [inline src/common/projects/Libraries.js:135]:
```suggestion
              target="_blank"
              rel="noopener nofererrer"
```

File: src/common/projects/Libraries.js [hunk 4]
```diff
@@ -99,16 +110,59 @@ const LibraryCard = ({name, logo, libraries}) => {
...
+              rel="noopener nofererrer"
```
Comment (dthaler) [inline src/common/projects/Libraries.js:136]:
```suggestion
              rel="noopener noreferrer"
```

**Review by dthaler — CHANGES_REQUESTED** [review]

**Review by xmulligan — APPROVED** [review]

**Pull request merged**